app/forms/idv/state_id_form.rb

```diff
@@ -28,7 +28,7 @@ def submit(params)
       cleaned_errors.delete(:identity_doc_address2, :nontransliterable_field)
 
       FormResponse.new(
-        success: valid?,
+        success: validation_success,
```

Comment:
Since you're not making a functional change to this file, I recommend excluding this from the PR.

Reply:
I see, good catch then. Both should be fixed; I think this was an oversight during some of the prior work on these errors. What do you think about updating the tests to validate this change?

Reply:
The state_id_form_spec test is newly added and does test this. Let me check the address_form test.

Reply:
address_form has no spec test, so we can add it.

Reply:
@NavaTim, made change to the address form and added test.